docs/about/all/support/guidelines-for-gdpr-compliance.md

```diff
@@ -11,32 +11,28 @@ redirect_from:
 
 {% info_block errorBox %}
 
-This document conforms to the [DSGVO](https://de.wikipedia.org/wiki/Datenschutz-Grundverordnung) and [RGPD](https://fr.wikipedia.org/wiki/R%C3%A8glement_g%C3%A9n%C3%A9ral_sur_la_protection_des_donn%C3%A9es) rules.
+This document conforms to the GDPR.
```

Comment:
Any particular reason we need to mention this?

Reply:
Answered in - https://spryker.slack.com/archives/C3KMWM1UZ/p1713176881720259?thread_ts=1712141501.987429&cid=C3KMWM1UZ